DeAceroDLabs/marketplace-frontend
# fix: [DMP-631] Hide description if it's null

## What is done

[[DMP-631] Hide description section if it's empty](https://deacero.atlassian.net/browse/DMP-631)

## How is done

1. Added a validation on Product Detail to hide the description section if this is empty.

## How to test

1. Run the project via `npm start`
2. Describe step by step
3. How to test your feature
4. Cover all the possible scenarios 
5. That you want to the team to notice

## Evidence

Add images or screen  recordings on how the feature was working after and before the fix (if applies)
